html/modules/custom/hr_paragraphs/templates/hr-paragraphs-rss-feed.html.twig

```diff
@@ -4,7 +4,8 @@
       <li class="external-feed--item">
       <div class="external-feed--item-title">
         <h3>
-          {% if options.display_date %}{{ item.date|date('d-M-Y') }} - {% endif %}<a rel="noopener noreferrer external nofollow" class="external-feed--item-link" href=" {{ item.link }}">{{ item.title }}</a>
+          {% if options.display_date %}{{ item.date|date('d-M-Y') }} - {% endif %}
+          <a rel="{% if 'reliefweb.int' not in item.link %}noopener noreferrer external nofollow{% endif %}" class="external-feed--item-link" href="{{ item.link }}">{{ item.title }}</a>
```

Comment:
maybe safer to use `{% if not (item.link starts with 'https://reliefweb.int') %} otherwise links as `https://example.com/reliefweb.int/bad' will slip trough

Reply:
yeah. I'd noted that in the ticket but figured the likelihood of an approved editor doing that was really low. Still, better safe than sorry 👍